java/opendcs/src/main/java/decodes/tsdb/algo/RelativeHumidity.java

```diff
@@ -12,34 +12,27 @@
 public class RelativeHumidity extends AW_AlgorithmBase
 {
 
-	@Input
```

Comment:
Our standard is to have these on the line above even for variables.

Sorry, we need to actually provide a proper style config to make this easier on people.